### PR TITLE
Solve Picard space issue

### PR DESCRIPTION
**Pull request by marinafloresp**
# Description

This pull request will update scripts that use Picard software. Specifying an alternative temporary directory in the bashrc file is not enough for the software. Picard needs a temporary directory to be specified in the command line with argument `-TMP_DIR` when large samples are input to avoid running out of space. This temporary directory can be specified either in the .bashrc file or in the config.txt file of the pipeline as a variable: `TMPDIR`. 

In addition, some scripts have been updated.

## Issue ticket number

This pull request is to address issue: #260

## Type of pull request

- [X] Bug fix
- [ ] New feature/enhancement
- [ ] Code refactor
- [X] Documentation update

## Checklist

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have tested my code to check that it is functional
- [ ] I have used linters to check for common sources of errors
- [ ] I have implemented fail safes in my code to account for edge cases
- [X] I have made the corresponding changes to the documentation

### CODE REVIEW

File: sequencing/ATACSeq/subScripts/compareBamWithGenotypes.sh [hunk 1]
```diff
@@ -57,9 +57,9 @@ then
     fi
     
     samtools addreplacerg -r "@RG\tID:RG1\tSM:SampleName\tPL:Illumina\tLB:Library.fa" -o ${ALIGNED_DIR}/${sampleName}_sorted_name.bam ${ALIGNED_DIR}/${sampleName}_sorted.bam
-
+    
     ## mark duplicates only
-    picard MarkDuplicates -INPUT ${ALIGNED_DIR}/${sampleName}_sorted_name.bam -OUTPUT ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup.bam -METRICS_FILE ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_metrics.txt
+    picard MarkDuplicates -INPUT ${ALIGNED_DIR}/${sampleName}_sorted_name.bam -OUTPUT ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_dedup.bam -METRICS_FILE ${ALIGNED_DIR}/baseRecalibrate/${sampleName}_metrics.txt -TMP_DIR ${TMPDIR}
```
Comment (sof202) [inline sequencing/ATACSeq/subScripts/compareBamWithGenotypes.sh:62]: Should this be `--` or `-`?

File: sequencing/ATACSeq/subScripts/alignment.sh [hunk 2]
```diff
@@ -88,7 +88,8 @@ rm ${ALIGNED_DIR}/${sampleName}_noMT.bam
 # Mark duplicates
 samtools addreplacerg -r "@RG\tID:RG1\tSM:SampleName\tPL:Illumina\tLB:Library.fa" -o ${ALIGNED_DIR}/${sampleName}_filt.name.bam ${ALIGNED_DIR}/${sampleName}.filt.bam
 
-picard MarkDuplicates -INPUT ${ALIGNED_DIR}/${sampleName}_filt.name.bam -OUTPUT ${ALIGNED_DIR}/${sampleName}.filt.dupmark.bam -METRICS_FILE ${ALIGNED_DIR}/${sampleName}_dupMetrics.txt -VALIDATION_STRINGENCY LENIENT -ASSUME_SORTED true -REMOVE_DUPLICATES false
+picard MarkDuplicates -INPUT ${ALIGNED_DIR}/${sampleName}_filt.name.bam -OUTPUT ${ALIGNED_DIR}/${sampleName}.filt.dupmark.bam -METRICS_FILE ${ALIGNED_DIR}/${sampleName}_dupMetrics.txt \
+-VALIDATION_STRINGENCY LENIENT -ASSUME_SORTED true -REMOVE_DUPLICATES false --TMP_DIR ${TMPDIR}
```
Comment (sof202) [inline sequencing/ATACSeq/subScripts/alignment.sh:92]: should this be `-` or `--`?

Comment (marinafloresp) [reply sequencing/ATACSeq/subScripts/alignment.sh:92]: Doesn't seem to make a difference. I can update it to make it consistent.

Comment (sof202) [reply sequencing/ATACSeq/subScripts/alignment.sh]: Oh ok, just checking is all.

**Review by sof202 — APPROVED** [review]

**Pull request closed without merging**